package.json

```diff
@@ -4,7 +4,10 @@
   "displayName": "swifttasks",
   "description": "Provide common tasks for swift packages",
   "version": "0.0.2",
-  "repository": "aussiegeek/vscode-swifttasks",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/aussiegeek/vscode-swifttasks"
```

Comment:
Without this change VS Code displayed a warning `Relative image URLs require a repository with HTTPS protocol in the package.json` at `![vscode-swifttasks-preview](./assets/readme-preview.gif)`.